protocol/avdl/stellar1/local.avdl

```diff
@@ -335,6 +335,59 @@ protocol local {
     boolean pending;
   }
 
+  // Check but do not send a request.
+  // Arguments are optional unless marked [required] or otherwise described.
+  // However, a coherent payment is required in order for res.readyToSend to returned as true.
+  BuildRequestResLocal buildRequestLocal(
+    int sessionID,
+
+    // Recipient username, acctID, or assertion.
+    // Examples: "", "russel", "GBQTE2V7Y356TFBZL6YZ2PA3KIILNSAAQRV5C7MVWS22KQTS4EMK7I46"
+    string to,
+    boolean toIsAccountID, // [required]
```

Comment:
can you request from an account ID? If not don't need toIsAccountID

Reply:
by account ID you mean a stellar address right? I don't see that in design, so looks like at least not for now. So I'll remove it for now.